demo/README.md

```diff
@@ -153,6 +153,8 @@ In an another use-case, the FIDO IoT Manufacturer provisions the Device using DI
 
 8. Complete Transfer Ownership 1 and 2 (TO1 and TO2) by starting the FIDO IoT HTTP Java Device Sample.
 
+In a special case of use-case: 2, the owner generates the size 0 ownership voucher using reseller key. In such a scenario, resale can be performed using the same steps as listed above, except step 5. User doesn't need to perform step 5 as ownership will already be with the reseller in which case copying owner's key in reseller keystore will not be required. Use the setupinfo API listed in [Owner README](owner/README.md#fido-iot-owner-rest-apis) for the same.
```

Comment:
`using a different public key other than the one used for onboarding (say Owner2 key). In this particular implementation, Owner2 key is same as the keys used in Reseller instance.`

The description can be updated on above lines.

Reply:
Done